span.go

```diff
@@ -47,6 +35,22 @@ type Span interface {
 	// Suitable for serializing over the wire, etc.
 	TraceContext() TraceContext
 
+	// Event() is equivalent to
+	//
+	//   Log(time.Now(), LogData{Event: event})
+	//
+	// if payload is unspecified; otherwise, only one payload argument is
+	// accepted, and Event() is equivalent to
+	//
+	//   Log(time.Now(), LogData{Event: event, Payload: payload[0]})
+	//
+	Event(event string, payload ...interface{})
+
+	// Log() records `data` to this Span.
+	//
+	// See LogData for semantic details.
+	Log(data LogData)
```

Comment:
In languages that support overloading, id expect them to just add a timestamp arg and skip the intermediary.